python/paddle/nn/functional/vision.py

```diff
@@ -68,7 +72,7 @@ def affine_grid(theta, out_shape, align_corners=True, name=None):
                [ 0.03333333,  1.83333337],
                [-0.43333334,  2.23333335]]]])
     """
-    if not isinstance(theta, Variable):
+    if not isinstance(theta, (Variable, paddle.pir.OpResult)):
```

Comment:
```suggestion
    if not isinstance(theta, (Variable, paddle.pir.Value)):
```

目前 paddle 在做 python 端 OpResult 和 Value 的统一工作，以后统一使用 paddle.pir.OpResult

Reply:
收到